api/command.go

```diff
@@ -20,7 +20,7 @@ import (
 type CommandProvider interface {
 	GetTrigger() string
 	GetCommand(c *Context) *model.Command
-	DoCommand(c *Context, channelId string, message string) *model.CommandResponse
+	DoCommand(c *Context, post *model.Post, message string) *model.CommandResponse
```

Comment:
Yes, `model.Post` has many unnecessary parameter  in this case, so your suggestion is nice.
I'll implement your idea, and push later. Thanks!